### PR TITLE
feat(schema): add 2.2.0-pr1 version

### PR DESCRIPTION
**Pull request by macintoshpie**
### Additions
- Bumps all version references to 2.2.0-pr1, and adds it as a enum to version attribute.

### CODE REVIEW

**Comment by macintoshpie** [comment]
@nllong can you generate the documentation as well?

**Comment by macintoshpie** [comment]
Also, the CHANGELOG updates I added aren't too helpful. I didn't realize it'd be pulling the PR name and the tags. Want me to go back and edit the PRs and regenerate the changelog?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,51 @@
...
+| Reports         | 7  |
+| Systems         | 1  |
+| Validation         | 0  |
+| Other         | 2  |
+| **Total**      | 12 |
+
+| Change Type    | Count |
+|----------------|-------|
+| Breaking Change         | 0  |
+| Non-breaking Change         | 0  |
```
Comment (nllong) [inline CHANGELOG.md:26]: hmm. looks like the non-breaking change didn't work... should be 12, right?

Comment (macintoshpie) [reply CHANGELOG.md:26]: Oops, forgot to add those tags

**Review by nllong — APPROVED** [review]

**Comment by nllong** [comment]
Looks good, merging!

**Pull request merged**